05.ios/05.solving-problems/09.how-to-avoid-compatibility-issues-with-FaceTime/docs.en.md

```diff
@@ -0,0 +1,21 @@
+---
+title: 'How to avoid compatibility issues with FaceTime'
+taxonomy:
+    category:
+        - docs
+   
+---
+
+It turned out that Full-Tunnel mode can interfere not only with compatibility with other VPN applications, but also with FaceTime.
+
+Some users have encountered the problem that FaceTime does not work on the device when the AdGuard app for iOS is in Full-Tunnel mode.
+
+It is likely but not guaranteed that FaceTime will work when AdGuard is in Full-Tunnel mode without VPN icon (also incompatible with other VPN apps, unstable). However, if you want to use FaceTime and make sure that video/audio calls don't stop working, use Split-Tunnel mode.
```

Comment:
(also incompatible with other VPN apps, unstable) - what is unstable and incompatible? Full-Tunnel mode?